charts/kueue/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.11.4"
+appVersion: "v0.11.5"
```

Comment:
don't change it manually, it is bumped during the release process

Reply:
ooh ok, sorry for that.

Reply:
adjusted